### PR TITLE
Fix default value display of method arguments for GDScript documentation and Tooltips

### LINKED ISSUE #62933: GDScript comment documentation does not show default values for functions arguments
### Godot version

4.0.alpha11

### System information

Windows11, Nvidia RTX 2060, Vulkan

### Issue description

After commenting documentation for a function with argument(s) with a default value, the default value of the argument is not displayed in the documentation.

### Steps to reproduce

- Create some documentation comment for a function with argument(s) with a default value
- The default value of the argument is not displayed in the documentation.

### Minimal reproduction project

_No response_

### PR DESCRIPTION
**Pull request by Xwdit**
Fix #62933

### CODE REVIEW

File: modules/gdscript/gdscript.cpp [hunk 1]
```diff
@@ -285,7 +285,11 @@ void GDScript::_get_script_method_list(List<MethodInfo> *r_list, bool p_include_
 #endif
 				mi.arguments.push_back(arginfo);
 			}
-
+#ifdef TOOLS_ENABLED
+			for (int i = 0; i < func->get_default_arg_values().size(); i++) {
+				mi.default_arguments.push_back(func->get_default_arg_values()[i]);
+			}
```
Comment (akien-mga) [inline modules/gdscript/gdscript.cpp:291]: Doesn't this work?
```suggestion
			mi.default_arguments.append_array(func->get_default_arg_values());
```

**Comment by akien-mga** [comment]
Could you squash the commits? See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase) for instructions.

File: modules/gdscript/gdscript.cpp (hunk 1 above)
Comment (Xwdit) [reply modules/gdscript/gdscript.cpp:291]: Thanks for the reminder, this is indeed the more concise way

**Comment by Xwdit** [comment]
> Could you squash the commits? See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase) for instructions.

Thanks for the reminder, I squashed the commits

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**